OrbitCore/OrbitFunction.h

```diff
@@ -54,61 +45,34 @@ class Function {
            std::string_view file, uint32_t line);
 
   const std::string& Name() const { return name_; }
-  const std::string& PrettyName() const {
-    return pretty_name_.empty() ? name_ : pretty_name_;
-  }
-  std::string Lower() { return ToLower(PrettyName()); }
-  const std::string& GetLoadedModulePath() const { return loaded_module_path_; }
-  std::string GetLoadedModuleName() const {
-    return Path::GetFileName(loaded_module_path_);
+  const std::string& PrettyName() const { return pretty_name_; }
+  const std::string& LoadedModulePath() const { return loaded_module_path_; }
+  void SetLoadedModulePath(const std::string& loaded_module_path) {
+    loaded_module_path_ = loaded_module_path;
   }
-  void SetModulePathAndAddress(std::string_view module_path,
-                               uint64_t module_adddress) {
-    loaded_module_path_ = module_path;
-    module_base_address_ = module_adddress;
+  uint64_t ModuleBaseAddress() const { return module_base_address_; }
```

Comment:
here and elsewhere in this file - module_base_address() or GetModuleBaseAddress()

Reply:
I prefer the first option as it won't require changes when moving to protobuf.